lxc/delete.go

```diff
@@ -92,6 +92,12 @@ func (c *cmdDelete) Run(cmd *cobra.Command, args []string) error {
 
 	// Process with deletion.
 	for _, resource := range resources {
+		projectName := c.global.conf.Remotes[resource.remote].Project
+
+		if c.global.flagProject != "" {
```

Comment:
Why do we need this section and/or the line above? We should only be using the project name that is set in the client that is going to be used to delete the actual instance?